src/mocks/api/shopsResult.ts

```diff
@@ -0,0 +1,31 @@
+import { http, HttpResponse } from 'msw';
+import { API_PATH } from 'config/constants';
+import shopsResultData from './data/shopsResultData';
+
+interface RequestBody {
+  lat: number | undefined;
+  lng: number | undefined;
+}
+
+const handlers = [
+  http.post(`${API_PATH}/shops`, async ({ request }) => {
+    const body = await request.json() as RequestBody;
+
+    body.lat = 37.71;
+    body.lng = 127.97;
+
+    const url = new URL(request.url.toString());
+    const keyword = url.searchParams.get('keyword');
+
+    if (typeof body.lat !== 'number' || typeof body.lng !== 'number') {
+      return new HttpResponse(null, { status: 401 });
+    }
```

Comment:
별건 아니지만 api 상에서는 이 경우 500에러가 나네요..!
겸사겸사 에러 응답도 같이 긁어와 봤습니다!

```
const ERROR_RESPONSE = {
  "className": "HttpMessageNotReadableException",
  "errorMessage": "JSON parse error: Unrecognized token 'undefined': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false');",
  "code": 0,
  "httpStatus": "INTERNAL_SERVER_ERROR",
  "errorTrace": "서버쪽 어느 경로"
} as const; // 따로 위에 선언
```

```suggestion
    if (typeof body.lat !== 'number' || typeof body.lng !== 'number') {
      return new HttpResponse(errorResponse, { status: 500 });
    }
```

Reply:
500에러로 변경했습니다.[1141eab](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/117/commits/1141eab09c67ef5d62f21da7a3379c54b5333d63)